src/utils/componentsToHints.test.ts

```diff
@@ -0,0 +1,70 @@
+import componentsToHints from './componentsToHints';
+// @ts-expect-error
+import * as PropTypeComponents from '../../cypress/projects/themed/components';
+import * as TypeScriptComponents from '../../cypress/projects/typescript/components';
+
+describe('componentsToHints', () => {
+  it('should support javascript components with proptypes', () => {
+    const result = componentsToHints({
+      Foo: PropTypeComponents.Foo,
+      Bar: PropTypeComponents.Bar,
```

Comment:
```suggestion
      Bar: PropTypeComponents.Bar,
      Foo: PropTypeComponents.Foo,
```
Ditto for second test